src/hyp3_isce2/packaging.py

```diff
@@ -0,0 +1,476 @@
+    # First add ninety degrees to the azimuth angle to go from angle-from-east to angle-from-north,
+    # then convert to radians
+    azimuth_angle = ISCE2Dataset('los.rdr.geo', 'lv_phi', 2)
+    cmd = (
+        'gdal_calc.py '
+        f'--outfile {product_name}/{product_name}_{azimuth_angle.suffix}.tif '
+        f'-A merged/{azimuth_angle.name} --A_band={azimuth_angle.band} '
+        '--calc (90+A)*pi/180 --type Float32 --format GTiff --NoDataValue=0 '
+        '--creation-option TILED=YES --creation-option COMPRESS=LZW --creation-option NUM_THREADS=ALL_CPUS'
+    )
+    subprocess.run(cmd.split(' '), check=True)
+
+    ds = gdal.Open('merged/filt_topophase.unw.geo')
+    geotransform = ds.GetGeoTransform()
+    del ds
+
+    epsg = utm_from_lon_lat(geotransform[0], geotransform[3])
+    files = [str(path) for path in Path(product_name).glob('*.tif') if not path.name.endswith('rdr.tif')]
+    for file in files:
+        gdal.Warp(
+            file,
+            file,
+            dstSRS=f'epsg:{epsg}',
+            creationOptions=['TILED=YES', 'COMPRESS=LZW', 'NUM_THREADS=ALL_CPUS'],
+            xRes=pixel_size,
+            yRes=pixel_size,
+            targetAlignedPixels=True,
+        )
+
+
+def convert_raster_from_isce2_gdal(input_image, ref_image, output_image):
+    """Convert the water mask in WGS84 to be the same projection and extent of the output product.
+
+    Args:
+        input_image: dem file name
+        ref_image: output geotiff file name
+        output_image: water mask file name
+    """
+
+    ref_ds = gdal.Open(ref_image)
+
+    gt = ref_ds.GetGeoTransform()
+
+    pixel_size = gt[1]
+
+    minx = gt[0]
+    maxx = gt[0] + gt[1] * ref_ds.RasterXSize
+    maxy = gt[3]
+    miny = gt[3] + gt[5] * ref_ds.RasterYSize
+
+    crs = ref_ds.GetSpatialRef()
+    epsg = CRS.from_wkt(crs.ExportToWkt()).to_epsg()
+
+    del ref_ds
+
+    gdal.Warp(
+        output_image,
+        input_image,
+        dstSRS=f'epsg:{epsg}',
+        creationOptions=['TILED=YES', 'COMPRESS=LZW', 'NUM_THREADS=ALL_CPUS'],
+        outputBounds=[minx, miny, maxx, maxy],
+        xRes=pixel_size,
+        yRes=pixel_size,
+        targetAlignedPixels=True,
+    )
+
+
+def water_mask(unwrapped_phase: str, water_mask: str) -> None:
+    """Apply the water mask to the unwrapped phase
+
+    Args:
+        unwrapped_phase: The unwrapped phase file
+        water_mask: The water mask file
+    """
+
+    convert_raster_from_isce2_gdal('water_mask.wgs84', unwrapped_phase, water_mask)
+    cmd = (
+        'gdal_calc.py '
+        f'--outfile {unwrapped_phase} '
+        f'-A {unwrapped_phase} -B {water_mask} '
+        '--calc A*B '
+        '--overwrite '
+        '--NoDataValue 0 '
+        '--creation-option TILED=YES --creation-option COMPRESS=LZW --creation-option NUM_THREADS=ALL_CPUS'
+    )
+    subprocess.run(cmd.split(' '), check=True)
+
+
+def make_readme(
```

Comment:
On second thought, we should hold of on making any changes in the readme/parameter file until we standardize them for single and multi-burst InSAR (multi-burst InSAR full release). Otherwise, the readme and parameter file templates will diverge.